addons/mail/static/src/models/attachment_viewer/attachment_viewer.js

```diff
@@ -20,6 +22,71 @@ function factory(dependencies) {
                 dialog.delete();
             }
         }
+
+        /**
+         * Handle onclick on next button
+         *
+         * @param {MouseEvent} ev
+         */
+        onClickNext(ev) {
+            markEventHandled(ev, 'attachmentViewer.clickNext');
```

Comment:
I would use uppercase for the first letter here, as it is the model name which is a class.

check all of them for this